src/components/Loader/People/People.tsx

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable @typescript-eslint/indent */
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../../types';
+import { getPeople } from '../../../api';
+import { Link, useLocation } from 'react-router-dom';
+
+export const People = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState('');
+
+  useEffect(() => {
+    setLoading(true);
+    getPeople()
+      .then(data => setPeople(data))
+      .catch(() => setError('Something went wrong'))
+      .finally(() => setLoading(false));
+  }, []);
+
+  const peopleNames = people.map(person => person.name);
+  const location = useLocation();
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+      <div className="block">
+        <div className="box table-container">
+          {loading && <Loader />}
+
+          {error && !loading && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              {error}
+            </p>
+          )}
+
+          {!people.length && !loading && (
+            <p data-cy="noPeopleMessage">There are no people on the server</p>
+          )}
+
+          {!loading && (
+            <table
+              data-cy="peopleTable"
+              className="table is-striped is-hoverable is-narrow is-fullwidth"
+            >
+              <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Sex</th>
+                  <th>Born</th>
+                  <th>Died</th>
+                  <th>Mother</th>
+                  <th>Father</th>
+                </tr>
+              </thead>
+
+              <tbody>
+                {people.map(person => {
+                  const isShe = person.sex === 'f';
+
+                  return (
+                    <tr
+                      data-cy="person"
+                      key={person.name}
```

Comment:
Using `person.name` as a key might not be unique if there are people with the same name. Consider using a unique identifier like `person.id` if available.